lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -17,6 +17,20 @@
     </div>
 </div>
 
+<div class="wrapper-message">
+    <div class="message">
+        <p>
+            <%= gettext("Hey, can you see this? Instructor toolbar placeholder") %>
```

Comment:
This should be `<%- gettext("...") %>` in order to allow translators to use HTML characters.